pages/1.11/administering-clusters/replacing-a-master-node/index.md

```diff
@@ -4,43 +4,44 @@ navigationTitle:  Replace a master node
 title: Replace a master node
 menuWeight: 800
 excerpt: Replacing a master node in an existing DC/OS cluster
-
 enterprise: true
 ---
-
 <!-- This source repo for this topic is https://github.com/mesosphere/dcos-docs-site -->
+You can replace a master node in an existing DC/OS cluster. You should keep in mind, however, that you should only ever replace one master at a time. The following steps summarize how to replace a master node for a DC/OS cluster.
 
+# To replace a master node
+1. Back up ZooKeeper state information using Exhibitor, the Guano utility, or a custom script.
 
-You can replace a master node in an existing DC/OS cluster. Only one master should be replaced at a time.
-
-### Remove the existing master
-
-To begin, simply shut down the master node you want to replace.
-
-### Add the new master
+    For example, if you download and extract the Guano utility, you can run a command similar to the following on the master node:
 
-The procedure for adding the new master node to replace the one taken offline in the previous step is quite simple.
+    ```bash
+    dcos-shell java -jar guano-0.1a.jar -u super -p secret -d / -o /tmp/mesos-zk-backup -s $ZKHOST:2181 && tar -zcvf zkstate.tar.gz /tmp/mesos-zk-backup/
+    ```
 
-#### `master_discovery: static`
+1. Back up the DC/OS identity and access management CockroachDB database to a file by running a command similar to the following on the master node:
```

Comment:
Similar to previous note, we should just link to https://docs.mesosphere.com/1.11/installing/installation-faq/#q-how-do-i-backup-the-iam-database-enterprise